.tekton/collector-component-pipeline.yaml

```diff
@@ -230,32 +230,199 @@ spec:
+      value: $(params.hermetic)
+    - name: PREFETCH_INPUT
+      value: $(params.prefetch-input)
+    - name: IMAGE_EXPIRES_AFTER
+      value: $(params.image-expires-after)
+    - name: COMMIT_SHA
+      value: $(tasks.clone-repository.results.commit)
+    - name: TARGET_STAGE
+      value: $(params.build-target-stage)
+    - name: BUILD_ARGS
       value:
-      - konflux-$(params.revision)
-      # TODO(ROX-24402): don't publish multiple tags when ability to install Slim collector is gone.
-      - $(tasks.determine-image-tag.results.IMAGE_TAG)-latest
-      - $(tasks.determine-image-tag.results.IMAGE_TAG)-slim
+      - COLLECTOR_TAG=$(tasks.determine-image-tag.results.IMAGE_TAG)
+    - name: SOURCE_ARTIFACT
+      value: $(tasks.prefetch-dependencies.results.SOURCE_ARTIFACT)
+    - name: CACHI2_ARTIFACT
+      value: $(tasks.prefetch-dependencies.results.CACHI2_ARTIFACT)
+    - name: ACTIVATION_KEY
+      value: subscription-manager-activation-key
+    - name: PLATFORM
+      value: linux/arm64
     taskRef:
       params:
       - name: name
-        value: apply-tags
+        value: buildah-remote-oci-ta
       - name: bundle
-        value: quay.io/redhat-appstudio-tekton-catalog/task-apply-tags:0.1@sha256:29add9a49a2281a3755a9b580d2b9c5cb110231b14cccf8ade2fd7895a9b4b4a
+        value: quay.io/redhat-appstudio-tekton-catalog/task-buildah-remote-oci-ta:0.2@sha256:b62eadffad6737b237cbfa3e78114daba18da17e5028fe71707388cf8a102f31
       - name: kind
         value: task
       resolver: bundles
+    when:
+    - input: $(tasks.init.results.build)
+      operator: in
+      values: [ "true" ]
+    timeout: 1h30m0s
+
+  - name: build-image-manifest
+    params:
+      - name: IMAGE
+        value: $(params.output-image-repo):$(tasks.determine-image-tag.results.IMAGE_TAG)
+      - name: COMMIT_SHA
+        value: $(tasks.clone-repository.results.commit)
+      - name: IMAGES
+        value:
+          - $(tasks.build-container-amd64.results.IMAGE_REF)
+          - $(tasks.build-container-s390x.results.IMAGE_REF)
+          - $(tasks.build-container-ppc64le.results.IMAGE_REF)
+          - $(tasks.build-container-arm64.results.IMAGE_REF)
+    taskRef:
+      params:
+        - name: name
+          value: build-image-manifest
+        - name: bundle
+          value: quay.io/konflux-ci/tekton-catalog/task-build-image-manifest:0.1@sha256:fd0a0cf019621d6b577f1b9ab774bb1832f7cba61b4ceee2fd1bffc96895abf9
+        - name: kind
+          value: task
+      resolver: bundles
+    when:
+      - input: $(tasks.init.results.build)
+        operator: in
+        values: [ "true" ]
+
+  - name: build-image-manifest-extra
+    matrix:
+      params:
+        - name: IMAGE
+          value:
+            - $(params.output-image-repo):$(tasks.determine-image-tag.results.IMAGE_TAG)-latest
+            - $(params.output-image-repo):$(tasks.determine-image-tag.results.IMAGE_TAG)-slim
+            - $(params.output-image-repo):konflux-$(params.revision)
```

Comment:
[nit] Didn't know this was a thing! We could've used a single manifest build for all components.

Anyways, I don't see this as a blocker, but maybe we could unify manifest creation on follow up PRs.